.github/workflows/graphqlWatermelonTest.yaml

```
@@ -0,0 +1,89 @@
+name: GraphQL authorization
+
+on:
+  schedule:
+  - cron:  '45 16 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check without bearer token
+        id: queryNoToken
```

```suggestion
        id: query-no-token
```

Just for consistency with the other pipelines. Same for the other id